StampIcon/Stamper.swift

```diff
@@ -53,8 +49,8 @@ struct Stamper {
             
             var badgeRect = rect.center(CGSize(width: maxWidth * 2, height: textSize.height + kVerticalPadding))
             
-            badgeRect.origin.x += (1/6) * maxWidth
-            badgeRect.origin.y -= (1/6) * maxHeight
+            badgeRect.origin.x += (1.0/6) * maxWidth
+            badgeRect.origin.y -= (1.0/6) * maxHeight
```

Comment:
Note that this changes behaviour, always evaluated to zero before. I haven't tested :grimacing: